features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -896,8 +896,8 @@
     <feature name="openhab-misc-mqttbroker" description="Embedded MQTT Broker" version="${project.version}">
         <feature>openhab-runtime-base</feature>
         <feature>openhab-transport-mqtt</feature>
-        <requirement>esh.tp;filter:="(feature=commons-net)"</requirement>
-        <feature dependency="true">esh.tp-commons-net</feature>
+        <requirement>openhab.tp;filter:="(feature=commons-net)"</requirement>
+        <feature dependency="true">openhab.tp-commons-net</feature>
```

Comment:
Is this some special version of commons-net or do we need to add this requirement/feature to each feature that depends on commons-net?

There are a few issues (#4574, #4709, #3812) where users install the "compact" package on initial installation and can't install bindings using commons-net. 

So perhaps this should also be added to:

* denonmarantz
* dscalarm
* lutron
* max
* network
* ntp
* opensprinkler
* plclogo
* russound
* samsungtv
* zoneminder
* zway

Maybe it makes sense to install commons-net by default instead of adding it to each binding feature that uses it? We seem to do  the same for other libraries.

Reply:
> Maybe it makes sense to install commons-net by default

I think it actually has been the case in the past (or maybe it was only the case for the compat layer, which almost everybody had installed). 
I have created https://github.com/openhab/openhab-core/pull/545, which should hopefully fix the issues you mentioned above.

Reply:
Great. Then I will remove it here.